config/initializers/health_monitor.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+Rails.application.config.after_initialize do
+  HealthMonitor.configure do |config|
+    config.cache
+
+    config.path = :health
+
+    config.file_absence.configure do |file_config|
+      file_config.filename = "public/remove-from-nginx"
+    end
+
+    config.error_callback = proc do |e|
+      Rails.logger.error "Health check failed with: #{e.message}"
+      Honeybadger.notify(e) unless e.is_a(HealthMonitor::Providers::FileAbsenceException)
```

Comment:
As a team we've decided to remove the Honeybadger notifications for the health checks - would you be ok with taking them off in this PR?
```suggestion
```